createdisk.sh

```diff
@@ -29,10 +29,22 @@ function create_disk_image {
     rm -fr $tarballDirectory/${VM_PREFIX}-master-0 $tarballDirectory/${VM_PREFIX}-base
 }
 
+function update_json_description {
+    cat $1/crc-bundle-info.json \
+        | ${JQ} ".clusterInfo.masterHostname = \"${VM_PREFIX}-master-0\"" \
```

Comment:
@cfergeau you added it here but your sample file still doesn't have a field for `masterHostname` under `clusterInfo`

Reply:
Ah indeed, thanks for catching this!